README.md

```diff
@@ -875,6 +875,83 @@ if (command === 'hello') {
 }
 ```
 
+.locale(locale)
+---------------
+
+Set a locale other than the default `en` locale:
+
+```js
+var argv = require('./')
```

Comment:
This should probably be `require('yargs')` instead of `require('./')`. :smiley_cat: